app/views/play/CampaignView.coffee

```diff
@@ -94,10 +93,12 @@ module.exports = class CampaignView extends RootView
             if reward not in earned[group]
               console.warn 'Filling in a gap for reward', group, reward
               earned[group].push(reward)
-
     @supermodel.loadCollection(@earnedAchievements, 'achievements', {cache: false})
 
-    @listenToOnce @campaign, 'sync', @getLevelPlayCounts
+    @campaign = new Campaign({_id:@terrain})
+    @listenToOnce @campaign, 'sync', @onCampaignLoaded
+    @campaign = @supermodel.loadModel(@campaign).model
```

Comment:
I think we should declare the listener first and call loadModel() later because there would be a (super) rare case that loadModel() runs and finishes before we could declare onSync listener
I dont think this case would likely to happen but it could happen